Forge/src/main/java/me/ferdz/placeableitems/init/PlaceableItemsItemsRegistry.java

```diff
@@ -8,11 +8,13 @@
 import net.minecraftforge.fml.common.Mod;
 
 @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD)
-public class PlaceableItemsItemsRegistry {
+public final class PlaceableItemsItemsRegistry {
```

Comment:
While you're at it, could you fix this class name by renaming it to `PlaceableItemsItemRegistry` ?